### DISCUSSION TITLE
Add a link in the dropdown menu for Actions

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#13336] [Unsupported Versions] [answered]
I have a website that looks up IP addresses. How can I add that to the Actions on the dropdown meu that displays when you click a row?

**Comment by dougburks** [comment, marked as answer]
Please see the documentation:
https://docs.securityonion.net/en/2.4/soc-customization.html#action-menu

**Reply by ghost** [reply]
Awesome. Thank you!

**Reply by dougburks** [reply]
We plan to make this even easier in the next version:
https://github.com/Security-Onion-Solutions/securityonion/issues/13346

**Reply by ghost** [reply]
I seem to remember in an older version that I had to edit a text file in an editor. So this is great.

**Reply by Carlos-mb** [reply]
Hi
I guess there is a mistake in documentation:

```
Insert the following as the next to last line:

,{ "name": "AbuseIPDB", "description": "Search for this value at AbuseIPDB", "icon": "fa-external-link-alt", "target": "_blank","links": [ "https://www.abuseipdb.com/check/{value}" ]}
```
I think the 1st comma shouldn't be there. 

Regards,
Carlos

**Reply by dougburks** [reply]
This will be fixed in the next release.

**Reply by Carlos-mb** [reply]
Thanks. 

I've playing a bit with this feature and I haven't found a chance to make the menu option appear only for IP address (for example). 

I have added this:

{ "name": "Whois", "description": "Search for this value at bgp.he.net", "icon": "fa-external-link-alt", "target": "_blank","links": [ "https://bgp.he.net/ip/{value}#_whois" ]}

It should be great to make it appear only when {value} is an IP. I have deduced than other actions doesn't appear when they use a field that is not available (that's a great point), but I can't use that chance because whois make sense for all IP fields in the event.

Regards.
